client-java/controller/src/main/java/org/evomaster/client/java/controller/problem/rpc/schema/params/ObjectParam.java

```diff
@@ -256,13 +256,13 @@ public List<String> newInstanceWithJava(boolean isDeclaration, boolean doesInclu
             if (f.accessibleSchema != null && f.accessibleSchema.setterMethodName != null){
                 String fName = ownVarName;
                 boolean fdeclar = false;
-                if (f instanceof ObjectParam || f instanceof MapParam || f instanceof CollectionParam || f instanceof DateParam || f instanceof  BigDecimalParam || f instanceof BigIntegerParam){
+                if (f instanceof ObjectParam || f instanceof MapParam || f instanceof CollectionParam || f instanceof DateParam || f instanceof  BigDecimalParam || f instanceof BigIntegerParam || f instanceof Protobuf3ByteStringParam){
                      fName = varName+"_"+f.getName();
                      fdeclar = true;
                 }
                 codes.addAll(f.newInstanceWithJava(fdeclar, true, fName, indent+1));
 
-                if (f instanceof ObjectParam || f instanceof MapParam || f instanceof CollectionParam || f instanceof DateParam || f instanceof  BigDecimalParam || f instanceof BigIntegerParam){
+                if (f instanceof ObjectParam || f instanceof MapParam || f instanceof CollectionParam || f instanceof DateParam || f instanceof  BigDecimalParam || f instanceof BigIntegerParam || f instanceof Protobuf3ByteStringParam){
```

Comment:
looks like a duplicated check in a `if`? if so, could refactor out in a function